### ISSUE TITLE
Audit not working audit_out_dir is not /var/tmp

### ISSUE THREAD

**Issue by pavloos** [ansible-lockdown/RHEL8-CIS#186]
**Describe the Issue**
It is not possible to set [audit_out_dir ](https://github.com/ansible-lockdown/RHEL8-CIS/blob/e1b72903db64398f90da504c479938ad92bd9c6b/defaults/main.yml#L694) to something else than the default `/var/tmp` because of two things:
1. `/var/tmp` mount should have `noexec` and it prevents execution of [run_audit.sh](https://github.com/ansible-lockdown/RHEL8-CIS/blob/devel/tasks/post_remediation_audit.yml#L4)
2. `run_audit.sh` [hardcodes](https://github.com/ansible-lockdown/RHEL8-CIS-Audit/blob/devel/run_audit.sh#L22) `/var/tmp` as a base location of `goss.yml` file.

**Expected Behavior**
Audit can be run.

**Actual Behavior**
WIth changed `audit_out_dir` the task fails with 

`WARNING - the /var/tmp/RHEL8-CIS-Audit/goss.yml is not available` 

```
TASK [cis : Pre Audit | Run pre_remediation RHEL8-CIS audit] ************************************************************************************************************************************************************
fatal: [default]: FAILED! => {"changed": true, "cmd": "/opt/build_audit/RHEL8-CIS-Audit/run_audit.sh -v /opt/build_audit/RHEL8-CIS-Audit/vars/ip-10-8-126-10.yml -o /opt/build_audit/ip-10-8-126-10_pre_scan_1651069043.documentation -g ['ungrouped']", "delta": "0:00:00.018752", "end": "2022-04-27 14:17:58.932785", "msg": "non-zero return code", "rc": 1, "start": "2022-04-27 14:17:58.914033", "stderr": "", "stderr_lines": [], "stdout": "\n## Pre-Checks Start\n\nOK Audit binary /usr/local/bin/goss is available\nWARNING - the /var/tmp/RHEL8-CIS-Audit/goss.yml is not available\n## Pre-checks failed please see output", "stdout_lines": ["", "## Pre-Checks Start", "", "OK Audit binary /usr/local/bin/goss is available", "WARNING - the /var/tmp/RHEL8-CIS-Audit/goss.yml is not available", "## Pre-checks failed please see output"]}
```

**Control(s) Affected**
Audit is affected.

**Environment (please complete the following information):**
 - Ansible Version:  2.12.4
 - Host Python Version: none
 - Ansible Server Python Version: 3.10.4
 - Using branch: 2.0.0
 - Additional Details:

**Additional Notes**
Anything additional goes here

**Possible Solution**
There is https://github.com/ansible-lockdown/RHEL8-CIS/blob/e1b72903db64398f90da504c479938ad92bd9c6b/defaults/main.yml#L700 but it is not used and maybe there is/was plan to use it somehow.

The solution I have is
https://github.com/pavloos/RHEL8-CIS-Audit/commit/f8a28f916113964e71075eb6e7c795184ea4bb1f
https://github.com/pavloos/RHEL8-CIS/commit/0b9345322a026667ab2d50738b5c8b3b0d5831df

**Comment by uk-bolly** [comment]
hi @pavloos 

Thank you for taking the time to feedback on this, this is a great catch. Even better to hear you are using it.
We have been using it partially as a test on a clean build or skipping in the workflow. But the suggestions you have are a great step forward.
I will look to introduce them over the coming weeks unless you wish to raise a PR in the meantime (it will need to be signed and GPG).
We would be looking to release the Audit just before the Remediate side of things.

Best regards

uk-bolly

**Comment by pavloos** [comment]
Hi @uk-bolly 

No worries at all! Thanks for maintaining this stuff, it's great.  PRs will there shortly.

cheers
Pawel

**Comment by uk-bolly** [comment]
hi @pavloos 

Great work and thanl you agian for taking the time to provide a solution and raise the PR.
Happy to to say this is now merged to devel.

Will leave until new main release but this should now be resolved.

Thank again

uk-bolly

**Comment by georgenalen** [comment]
This was addressed in release 2.1.0
